tests/test_ffimachine.py

```diff
@@ -0,0 +1,88 @@
+"""
+This contains a collection of functions to test the Machine API
+"""
+import os
+
+# import numpy as np
+import pytest
+from astropy.io import fits
+from astropy.table import Table
+from astropy.utils.data import get_pkg_data_filename
+
+from psfmachine import Machine, FFIMachine
+
+ffi_path = get_pkg_data_filename("./data/ffi_ch01_test.fits")
+ffi = FFIMachine.from_file(ffi_path, channel=1)
```

Comment:
We need to move this inside the function. If this fails, if it's not in a test function we won't understand why it failed.